doc/MONSTERS.md

```diff
@@ -24,25 +26,44 @@ Monster types support the following properties (mandatory, except if noted other
 "name": { "ctxt": "fish", "str": "pike", "str_pl": "pikes" }
 ```
 
-Name displayed in-game, and optionally the plural name and a translation context.
+Name displayed in-game, and optionally the plural name and a translation context (ctxt).
+
 If the plural name is not specified, it defaults to singular name + "s".
 
+Ctxt is used to help translators in case of homonyms (two different things with the same name). For example, pike the fish and pike the weapon.
+
 ## "description"
 (string)
 
 In-game description for the monster.
 
+## "categories"
+(array of strings, optional)
+
+Monster categories. Can be NULL, CLASSIC (only mobs found in classic zombie movies) or WILDLIFE (natural animals). If they are not CLASSIC or WILDLIFE, they will not spawn in classic mode.  One can add or remove entries in mods via "add:flags" and "remove:flags".
+
 ## "species"
 (array of strings, optional)
 
 A list of species ids. One can add or remove entries in mods via "add:species" and "remove:species", see Modding below. Properties (currently only triggers) from species are added to the properties of each monster that belong to the species.
 
 In mainline game it can be HUMAN, ROBOT, ZOMBIE, MAMMAL, BIRD, FISH, REPTILE, WORM, MOLLUSK, AMPHIBIAN, INSECT, SPIDER, FUNGUS, PLANT, NETHER, MUTANT, BLOB, HORROR, ABERRATION, HALLUCINATION and UNKNOWN.
 
-## "categories"
-(array of strings, optional)
+## "volume"
+(string)
 
-Monster categories. Can be NULL, CLASSIC (only mobs found in classic zombie movies) or WILDLIFE (natural animals). If they are not CLASSIC or WILDLIFE, they will not spawn in classic mode.  One can add or remove entries in mods via "add:flags" and "remove:flags".
+```JSON
+"volume": "40 L"
+```
+The numeric part of the string must be an integer.
+
+## "weight"
+(string)
+
+```JSON
+"weight": "3 kg"
+```
+The numeric part of the string must be an integer. Use the largest unit you can keep full precision with. For example: 3 kg, not 3000 g.
```

Comment:
I would add "accepts `g` and `kg` as units."

Reply:
This is fine to add this later, but shouldn't hold up this PR. I'll make a note to do that once this PR is complete.

Reply:
I don't know if "resolve conversation" is something that I should do or if @I-am-Erk should do, but I'm going to stab the button in the hopes that it gets this PR merged.